tools/dgeni/src/transforms/daffodil-api-package/index.ts

```diff
@@ -24,6 +25,7 @@ export const apiDocs =  new Package('checkout', [
   .processor(new CleanSelectorsProcessor())
   .processor(new MakeTypesHtmlCompatibleProcessor())
   .processor(new FilterOutPrivatePropertiesProcessor())
+  .processor(new AddInheritedDocsContentProcessor())
```

Comment:
Can this be done with the existing processor from dgeni? https://github.com/angular/dgeni-packages/commit/cf8b7444d527d5c97ee531d1dc71241413b3bdb7

Reply:
no. This PR is just for classes that implement interfaces and getting the docs for those interface members into the class. It looks like that existing processor is only for classes that extend each other. We should probably use that as well in the future though.